paper.qmd

```diff
@@ -50,6 +47,20 @@ Reconciling topologically different linestrings is a fundamental challenge in sp
 
 > "Due to the complexity and limitations of existing methods, planners and analysts often have to employ a heavily manual conflation process, which is time‐consuming and often prohibitively expensive." Lei, Ting; Lei, Zhen (2019).
 
+## Definitions
+
+The purpose of this section is to provide clear and concise definitions of terminologies used in the description of this algorithm. 
+
+We will use the simple feature access standards definitions of geometric primitives for the sake of consistency. In our algorithm we make extensive use of **Line** and **LineString** geometric primitives. A Line is defined by two **Point**s that are (x, y) coordinate pairs. A LineString is composed of 2 or more Points and "each consecutive pair of Points defines a Line segment." (@sfa FIXME cite). Each LineString is referred to as a **feature**. An array of features is referred to as a **FeatureCollection** (@geojson_spec). 
```

Comment:
Isn't the 'Line' word from the GeoRust people? May be work x-referencing them if so.

Reply:
No, actually it's from the simple feature standard! 
<img width="947" alt="image" src="https://github.com/nptscot/rnetmatch/assets/13534508/347fcdcf-05d9-4af2-aa62-86349bb0ee4f">

Reply:
Aha, we lack that in sf!